TBot/Program.cs

```diff
@@ -3730,12 +3730,13 @@ class Program {
 							.OrderBy(fleet => fleet.BackIn)
 							.ToList();
 					}
-
 					slots = UpdateSlots();
-					if (orderedFleets.Count == 0 || slots.ExpFree > 0) {
+					if (orderedFleets.Count == 0 || ((bool) settings.Expeditions.WaitForMajorityOfExpeditions && slots.ExpFree > (slots.ExpTotal / 2))) {
 						interval = Helpers.CalcRandomInterval(IntervalType.AboutFiveMinutes);
 					} else {
 						interval = (int) ((1000 * orderedFleets.First().BackIn) + Helpers.CalcRandomInterval(IntervalType.AMinuteOrTwo));
+						if (orderedFleets.Count >= 6)
```

Comment:
I chose a low value to be sure. In the event of high latency or with low internet speed, it is around 6/7 slots that the gap in the interval may appear.
Taking the value of the last return would amount to validating choosing `WaitForAllExpeditions`. Personally it makes sense because `WaitForMajorityOfExpeditions` allows you to ignore delays.

Reply:
i don't like that, i'd rather make that dynamically calculated. at the beginning 6 is not a low value